src/libstd/io/buffered.rs

```diff
@@ -322,7 +323,7 @@ impl<W> From<IntoInnerError<W>> for Error {
 }
 
 #[stable(feature = "rust1", since = "1.0.0")]
-impl<W: Send + fmt::Debug> error::Error for IntoInnerError<W> {
+impl<W: Reflect + Send + fmt::Debug> error::Error for IntoInnerError<W> {
```

Comment:
I have mixed feelings about this one. I decided to leave them as `Reflect` to avoid the `'static` restriction, but I don't feel too strongly either way. (FWIW, in the long run we will either stabilize `Reflect`, making this possible, or remove `Reflect`, making it irrelevant.)